persone.rb

```diff
@@ -1,8 +1,8 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id, :name, :age, :rentals
-  attr_accessor :classroom
+  attr_accessor :name, :age, :rentals
+  attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
```

Comment:
- [ ] There is a typo in this file's name  😨, Kindly rename it to `person.rb` instead of `persone.rb`to avoid the following error 🙏 

<img width="1018" alt="Screen Shot 2023-10-19 at 20 02 00" src="https://github.com/axcell-0/school-library/assets/102757126/a0143beb-5c0c-4c6e-b62e-e3b53b4574d9">